src/core/QueryManager.ts

```
@@ -829,17 +827,6 @@ export class QueryManager {
       const data = readQueryFromStore(readOptions);
       return maybeDeepFreeze({ data, partial: false });
     } catch (e) {
-      // next, try reading partial results, if we want them
```

We shouldn’t catch this error, correct? We should just let it throw. There isn’t much use in returning: `maybeDeepFreeze({ data: {}, partial: true })`.